scipy/stats/_continuous_distns.py

```diff
@@ -4242,6 +4247,53 @@ def _munp(self, n):
     def _entropy(self):
         return 2-np.log(2)
 
+    @_call_super_mom
+    @inherit_docstring_from(rv_continuous)
+    def fit(self, data, *args, **kwds):
+        if (isinstance(data, CensoredData)) or kwds.pop('superfit', False):
```

Comment:
```suggestion
    def fit(self, data, *args, **kwds):
        if kwds.pop('superfit', False):
```

Please double-check, but I don't think either of these are needed. 

1. Because `fit` here has no docstring (and because `_call_super_mom` uses `wraps` to avoid getting in the way), this `fit` gets the docstring from `rv_continuous` without the `inherit_docstring_from` decorator.  (TBH it doesn't look like `inherit_docstring_from` does anything unless the docstring has `%(super)s` in the text.) In any case, you can verify that `from scipy.stats import halflogistic; help(halflogistic.fit)` does the right thing with and without the `inherit_docstring_from` decorator.

2. The input to `fit` can't be a `CensoredData` because if it is, the `_call_super_mom` wrapper takes care of it.

Reply:
CircleCI was complaning in the last run before about this, that's why I added it. But I am getting more and more confused with the intricacies here. The CI failure from https://github.com/scipy/scipy/pull/18695#issuecomment-1599409680 also seems to be gone now. Feel free to push whatever is needed regarding the docstrings and censored data to this branch to get this over the finish line if you can find time. Once more thanks for your advice and thorough reviews.

Reply:
https://github.com/scipy/scipy/pull/18695#issuecomment-1599409680 was because https://github.com/scipy/scipy/pull/18695#discussion_r1233413979 hadn't been addressed. I suggested `inherit_docstring_from` and `extend_notes_in_docstring` to be considered depending on what you wanted to do with the docstring, not to just apply.

> Feel free to push whatever is needed regarding the docstrings and censored data to this branch to get this over the finish line if you can find time.

That would be as easy as pushing "Commit suggestion", but I would prefer not to here. (Every time I do, it slightly devalues the time already invested into review.) Do you plan on doing any more `fit` PRs? If so, I'd ask that you look into what's going on with these. Feel free to ask questions if you don't understand something.